pkg/broker/broker_test.go

```diff
@@ -1,43 +1,54 @@
-package kafka
+package broker
 
 import (
 	"github.com/spf13/cobra"
 	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
 
 	"github.com/confluentinc/kafka-rest-sdk-go/kafkarestv3"
 
 	"github.com/confluentinc/cli/v3/pkg/errors"
 )
 
-func (suite *KafkaClusterTestSuite) TestBroker_checkAllOrBrokerIdSpecified() {
+type KafkaClusterTestSuite struct {
+	suite.Suite
+}
+
+func (suite *KafkaClusterTestSuite) TestBroker_CheckAllOrIdSpecified() {
```

Comment:
This is a style of testing that we're trying to move away from. Can we use integration tests instead here?

Reply:
we have broker tests for --all flag. I'll just remove this one.!